src/itemviewformaction.cpp

```diff
@@ -203,8 +203,15 @@ bool ItemViewFormAction::process_operation(Operation op,
 		do_redraw = true;
 		break;
 	case OP_ENQUEUE: {
-		if (item->enclosure_url().length() > 0 &&
-			utils::is_http_url(item->enclosure_url())) {
+		if (item->enclosure_url().length() == 0) {
+			v->get_statusline().show_error(strprintf::fmt(
+					_("Item has no enclosures.")));
```

Comment:
There's no need for `strprintf::fmt` here, since there are no parameters to put into the string.